packages/cli/src/Server.ts

```diff
@@ -116,79 +116,36 @@ export class Server extends AbstractServer {
 
 	private async registerControllers() {
 		const { app } = this;
-
-		const controllers: Array<Class<object>> = [
-			EventBusController,
-			AuthController,
-			LicenseController,
-			OAuth1CredentialController,
-			OAuth2CredentialController,
-			OwnerController,
-			MeController,
-			DynamicNodeParametersController,
-			NodeTypesController,
-			PasswordResetController,
-			TagsController,
-			TranslationController,
-			UsersController,
-			SamlController,
-			SourceControlController,
-			WorkflowStatisticsController,
-			ExternalSecretsController,
-			OrchestrationController,
-			WorkflowHistoryController,
-			BinaryDataController,
-			VariablesController,
-			InvitationController,
-			VariablesController,
-			ActiveWorkflowsController,
-			WorkflowsController,
-			ExecutionsController,
-			CredentialsController,
-			AIController,
-			ProjectController,
-			RoleController,
-			CurlController,
-			UserSettingsController,
-		];
-
 		if (
 			process.env.NODE_ENV !== 'production' &&
 			Container.get(OrchestrationService).isMultiMainSetupEnabled
 		) {
-			const { DebugController } = await import('@/controllers/debug.controller');
-			controllers.push(DebugController);
+			await import('@/controllers/debug.controller');
 		}
 
 		if (isLdapEnabled()) {
 			const { LdapService } = await import('@/Ldap/ldap.service');
-			const { LdapController } = await require('@/Ldap/ldap.controller');
+			await require('@/Ldap/ldap.controller');
```

Comment:
Why `require` here?

Reply:
just oversight. should be an import. I'll update it.